frontend/public/components/utils/firehose.jsx

```diff
@@ -25,8 +25,12 @@ const processReduxId = ({k8s}, props) => {
   }
 
   if (!isList) {
-    const stuff = k8s.get(reduxID);
-    return stuff ? stuff.toJS() : {};
+    let stuff = k8s.get(reduxID);
+    if (stuff) {
+      stuff = stuff.toJS();
+      stuff.ignoreIfMissing = props.ignoreIfMissing;
```

Comment:
This should be removed since #1711 merged

Reply:
resolved by rebase